src/applications/vaos/components/calendar/CalendarContext.jsx

```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'react';
+
+export const CalendarContext = createContext(null);
```

Comment:
Interesting, react context is new to me so I'm not too confident in being able to confidently say these are the right changes. Hopefully someone else can chime in?

Reply:
I was mostly unfamiliar with it as well, I had to look it up. FWIU the useContext allows you to define some variables at the top of a tree of components and anything under that can get that value by calling useContext. It's like useState but it doesn't have to be in that specific component(it will search up). This way you can avoid passing props all the way down the tree multiple times and you don't have to go so far as to use redux. I think this is a good use for it.

I don't think we need to put this in it's own file though as I think it would be fine in the CalendarWidget.jsx component.